testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/epcpropagation/requiresnew/BikeManagerBean.java

```diff
@@ -67,8 +67,8 @@ public void runTest2() {
         try {
             bikeShop.forceRollback(2);
         } catch (RuntimeException ignore) {}
-        Assert.assertNotNull("extended persistence context must be associated with JTA tx during " +
-                "call to purchaseNowAndFlushDbChanges() and db changes saved when that method ends its JTA tx.", bikeRace.find(2));
+        Assert.assertNotNull("extended persistence context must be associated with Jakarta Transactions tx during " +
+                "call to purchaseNowAndFlushDbChanges() and db changes saved when that method ends its Jakarta Transactions tx.", bikeRace.find(2));
```

Comment:
`Jakarta Transactions tx`-> `Jakarta Transaction`

Reply:
@asoldano what about this one?

Reply:
"Jakarta Transactions transaction"

Reply:
Please apply the style choice from previous comments (either use `(Jakarta Transactions) transaction` or `Jakarta Transactions transaction`.